docs/core-concepts/validation-suites.mdx

```diff
@@ -0,0 +1,125 @@
+---
+sidebar_label: 'Validation Suites'
+---
+
+# Validation Suites
+
+**Validation Suites** help ensure your game is functioning as intended by verifying that specific criteria are met
+over the course of a [Gameplay Session](gameplay-sessions/gameplay-sessions-getting-started).
+A Validation Suite is composed of a collection of [Scenarios](#validation-scenarios)
+and [Rules](#validation-rules). Validation Suites are managed through a dashboard in the
+[Regression Games web interface](https://play.regression.gg/validations).
+
+## Creating a Validation Suite
+
+Create a Validation Suite from the [Validation Suites dashboard](https://play.regression.gg/validations)
+by clicking the **Create Suite** button at the top of the page.
+Enter a name and optional description for your Suite, and choose a Base Gameplay Session.
+This Gameplay Session will be used to create your Scenarios,
+which can then be run against other Gameplay Sessions in the future for regression testing.
+
+![Create a Validation Suite](img/validation-suites/create-validation-suite.gif)
+
+## Validation Scenarios
+
+A **Validation Scenario** defines a functional test case by asserting that specific actions are taken during
+a Gameplay Session and that they produce expected changes in the game state.
+Steps can assert that a game scene is loaded, wait for mouse and keyboard inputs, check for the existence
+of specific game objects, and much more.
+
+Validation Scenarios execute in the same order as they appear in the Validation Suite dashboard, but order does not impact how Scenarios run.
+Each Scenario starts at the first tick of the Gameplay Session, and a Suite continues to run remaining Scenarios when one fails.
+
+### Creating a Validation Scenario
+
+Open your Suite in the [Validation Suites dashboard](https://play.regression.gg/validations) and click the **Create Scenario** button.
+This opens the **Scenario Builder**, where you can view game state data and a screenshot for each tick in the Gameplay Session,
+and use this information to build steps for your Scenario.
+
+![Screenshot of a scenario](img/validation-suites/scenario-example.png)
```

Comment:
I think this is mostly self explanatory but at this point in the doc the reader doesn't know what each of the lines does. Having a sentence that explains what this scenario accomplishes can help them ground their assumptions in what it actually does, and potentially how it adds value. 

Maybe something like:
This Scenario does, x,y,z and can be used to ensure that a,b,c doesn't break as we make changes to our game.